### ISSUE TITLE
RAW images should be scene linear

### ISSUE THREAD

**Issue by gregcotten** [OpenImageIO/oiio#1944]
This is probably possible - I'm not really familiar with libraw but it looks like it *almost* returns normalized 0-1 scene linear data from RAW images?

**Comment by lgritz** [comment]
I'm not sure I understand the distinction. Are you suggesting to convert from values relative to the saturation limit of the sensor, into true photometric radiance units?

**Comment by gregcotten** [comment]
Yes, but I'm not sure if it's always possible without spectral sensitivity information about the camera. I'm going to do some research with AMPAS's [rawtoaces tool](https://github.com/ampas/rawtoaces).

**Comment by melMass** [comment]
You still need some kind of input transform.
Most camera vendors released them but it can't magically figure out the conversion from an arbitrary input.

**Comment by lgritz** [comment]
Was this a satisfactory answer? Or is something more needed?

**Comment by gregcotten** [comment]
I haven’t tested but if it works how you say that is perfect!

**Comment by lgritz** [comment]
I will close by amending my previous answer.  I understand now that the command I gave before will leave you in full ACES space, and it's likely you want it in ACEScg or some other "house" color space. That would look like this, assuming you have OCIO support:

    oiiotool -iconfig "raw:ColorSpace" ACES input.cr2 -colorconvert ACES ACEScg -o output_acescg.exr
